modules/op-creating-and-verifying-task-run-signatures-without-any-additional-authentication.adoc

```diff
@@ -86,5 +86,3 @@ $ tkn tr describe --last -o jsonpath="{.metadata.annotations.chains\.tekton\.dev
 $ cosign verify-blob --key k8s://openshift-pipelines/signing-secrets --signature ./signature ./payload
```

Comment:
Non-admin users will not have access to K8S secret, this could be updated to use only public key which can be shared with anybody.

Reply:
@PuneetPunamiya could you please comment? I can't craft any changes to commands myself, I don't understand them well enough.

Reply:
Ah that's a good point.

Reply:
It should be possible to run something like `cosign verify-blob --key /path/to/cosign.pub --signature ./signature ./payload`... that's a file that is created when running `cosign generate-key-pair k8s://openshift-pipelines/signing-secrets`. Since it's a public key, user who generated it can freely distribute it to whoever needs it.

Reply:
So instead of relying on `k8s://…` we should just point to a file (the same we used to create the secret, probably above in this doc)

Reply:
So overall the command will change from here https://github.com/openshift/openshift-docs/pull/59627/files/87767ef648631e392a477e70d48647b7f98074bd#diff-3e053f6f6c5181e4b8d2bad32827d736e80d293deee0e771d93f88c3d4b98eb8L77-L79

```bash=
export TASKRUN_UID=$(tkn tr describe --last -o  jsonpath='{.metadata.uid}')
tkn tr describe --last -o jsonpath="{.metadata.annotations.chains\.tekton\.dev/signature-taskrun-$TASKRUN_UID}" | base64 -d > sig
```

```bash=
cosign verify-blob --key </path/to/cosign.pub> --signature sig sig
```

Reply:
cc @vdemeester @ppitonak

Reply:
Changed, verification needed.

Reply:
@PuneetPunamiya @vdemeester I just realized that the new command set still uses cosign for verification, but @vdemeester mentioned cosign is not really officially supported. Is it possible to use skopeo here instead?

Reply:
> cosign verify-blob --key </path/to/cosign.pub> --signature sig sig

Listing `sig` twice is strange, IMHO the other command storing payload should still be in doc and would be provided as last arg of `cosign verify-blob`.

Reply:
Meeting held, issue resolved, commands pushed.